src/lexical-elements.rst

```diff
@@ -703,6 +704,112 @@ The :t:`type` of a :t:`raw byte string literal` of size ``N`` is ``&'static
    br#""#
    br##"left #"# right"##
 
+.. _fls_U1gHCy16emVe:
+
+C String Literals
+~~~~~~~~~~~~~~~~~
+
+.. rubric:: Syntax
+
+.. syntax::
+   CStringLiteral ::=
+       RawCStringLiteral
+     | SimpleCStringLiteral
+
+.. rubric:: Legality Rules
+
+:dp:`fls_VKCW830CzhhN`
+A :t:`c string literal` is a :t:`literal` that consists of multiple characters
+with an implicit 0x00 byte appended to it.
+
+:dp:`fls_XJprzaEn82Xs`
+The character sequence 0x0D 0xCA (carriage return, new line) is replaced by 0xCA
+(new line) inside of a :t:`c string literal`.
+
+.. _fls_p090c5oTnElW:
+
+Simple C String Literals
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. rubric:: Syntax
+
+.. syntax::
+
+   SimpleCStringLiteral ::=
+       $$c"$$ SimpleCStringContent* $$"$$
+
+   SimpleCStringContent ::=
+       AsciiEscape
+     | SimpleStringCharacter
+     | StringContinuation
+     | UnicodeEscape
+
+:dp:`fls_fnwQHo7twAom`
+A :t:`simple c string literal` is any :t:`Unicode` character except characters
+0x0D (carriage return), 0x22 (quotation mark), 0x5C (reverse solidus) and 0x00
+(null byte).
+
+.. rubric:: Legality Rules
+
+:dp:`fls_nPI7j0siGP8G`
+A :t:`simple c string literal` is a :t:`c string literal` where the characters are
+:t:`Unicode` characters.
+
+:dp:`fls_Ae7LM4Wg0NA7`
+The :t:`type` of a :t:`simple string literal` is :std:`&'static
+[core::ffi::CStr]`.
```

Comment:
Shouldn't this be `&'static core::ffi::CStr`?

Reply:
The `[]` is the processing syntax for the std linkage, it does render as `&'static core::ffi::CStr`

Reply:
...right :sweat_smile: